beacon_node/client/src/builder.rs

```diff
@@ -315,6 +315,9 @@ where
             client_config
                 .create_db_path()
                 .expect("unable to read datadir"),
+            client_config
+                .create_freezer_db_path()
+                .expect("unable to read freezer DB dir"),
```

Comment:
It seems like this `expect` should be handled via the `Result`. I'm not fussed if you thinks it's fine as-is :)

Reply:
Fixed